docs/spec/distribution/overview.md

```diff
@@ -2,71 +2,72 @@
+and validator proposer-reward pool. Due to the nature of passive accounting,
+whenever changes to parameters which affect the rate of reward distribution
+occurs, withdrawal of rewards must also occur.
 
- - Whenever withdrawing, one must withdraw the maximum amount they are entitled
-   too, leaving nothing in the pool. 
- - Whenever bonding, unbonding, or re-delegating tokens to an existing account, a
+- Whenever withdrawing, one must withdraw the maximum amount they are entitled
+   too, leaving nothing in the pool.
+- Whenever bonding, unbonding, or re-delegating tokens to an existing account, a
    full withdrawal of the rewards must occur (as the rules for lazy accounting
    change).
- - Whenever a validator chooses to change the commission on rewards, all accumulated 
+- Whenever a validator chooses to change the commission on rewards, all accumulated
    commission rewards must be simultaneously withdrawn.
 
 The above scenarios are covered in `hooks.md`.
 
-The distribution mechanism outlines herein is used to lazily distribute the
+The distribution mechanism outlines herein are used to lazily distribute the
 following rewards between validators and associated delegators:
- - multi-token fees to be socially distributed, 
- - proposer reward pool, 
- - inflated atom provisions, and
- - validator commission on all rewards earned by their delegators stake
+
+- multi-token fees to be socially distributed
+- proposer reward pool
+- inflated atom provisions
+- validator commission on all rewards earned by their delegators stake
 
 Fees are pooled within a global pool, as well as validator specific
 proposer-reward pools. The mechanisms used allow for validators and delegators
 to independently and lazily withdraw their rewards.  
 
-## Shortcomings 
+## Shortcomings
 
 As a part of the lazy computations, each delegator holds an accumulation term
 specific to each validator which is used to estimate what their approximate
-fair portion of tokens held in the global fee pool is owed to them. 
+fair portion of tokens held in the global fee pool is owed to them.
 
 ```
 entitlement = delegator-accumulation / all-delegators-accumulation
 ```
 
-Under the circumstance that there were constant and equal flow of incoming
+Under the circumstance that there were constant and equal flows of incoming
 reward tokens every block, this distribution mechanism would be equal to the
 active distribution (distribute individually to all delegators each block).
-However this is unrealistic so deviations from the active distribution will
+However, this is unrealistic so deviations from the active distribution will
 occur based on fluctuations of incoming reward tokens as well as timing of
-reward withdrawal by other delegators. 
+reward withdrawal by other delegators.
 
-If you happen to know that incoming rewards are about significantly move up,
+If you happen to know that incoming rewards are about significantly go up,
```

Comment:
about *to* significantly go up

Also, we could link to https://github.com/cosmos/cosmos-sdk/issues/2764, which is precisely this problem

Reply:
Updated.